internal/style/style_test.go

```diff
@@ -0,0 +1,39 @@
+package style_test
+
+import (
+	"testing"
+
+	"github.com/heroku/color"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/pack/internal/style"
+	h "github.com/buildpacks/pack/testhelpers"
+)
+
+func TestStyle(t *testing.T) {
+	color.Disable(true)
+	defer color.Disable(false)
```

Comment:
The style package uses colors pretty intensely, so it may be helpful to have some testing of the color output, though not in every case. Perhaps we can have one test case where we enable color, and ensure that it matches what we expect?

Reply:
sure, will do that